facebook/ThreatExchange
# Create datasets api

Summary
---------
previous PR : https://github.com/facebook/ThreatExchange/pull/516
Task : T88063259, this task is to create ThreatExchange setting page, This PR is to create delete api to be used in ThreatExchange setting page

Future Actions
---------
 Create UI page to use the delete api 

Test Plan
---------

1. black hmalib
2. mypy hmalib
3. python -m py.test to pass all tests
4. make docker
5. make upload_docker 
6.  terraform apply
7. test delete api in local and verify the result in DynamoDB

https://user-images.githubusercontent.com/81996660/116912102-c2f41980-ac15-11eb-8c99-89f0afd4b5c6.mov
